src/main/scala/gitbucket/core/controller/PullRequestsController.scala

```diff
@@ -338,14 +342,26 @@ trait PullRequestsControllerBase extends ControllerBase {
     }) getOrElse NotFound()
   })
 
+  post("/:owner/:repository/pull/:id/update_draft")(readableUsersOnly { baseRepository =>
+    (for {
+      issueId <- params("id").toIntOpt
+      (_, pullreq) <- getPullRequest(baseRepository.owner, baseRepository.name, issueId)
+      owner = pullreq.requestUserName
+      name = pullreq.requestRepositoryName
+      if hasDeveloperRole(owner, name, context.loginAccount)
```

Comment:
Adding constraint `hasDeveloperRole` means that only users who has write permission to the repository can release the draft status. This also means the author of pull request may not be able to do that. Is this ok?

Reply:
@takezoe Here the owner is the base repository owner, so the author of pull request can release the draft status. Also users having write permission can also release the draft status.